_docs/layout/navigation.md

```diff
@@ -233,116 +253,124 @@ navigational element.
 <div class="row">
 {% column left:"col-md-4" %}
--   Switching product lines is triggered via clicking the Product Button and
-    displays a dropdown element for selection
 
--   Leftside dropdowns provide links to product lines and NEVER include a
-    secondary CTA
+-   A user can switch product lines by clicking the product button,
+    then clicking a drop-down element.
 
--   Leftside dropdowns do not have a defined disabled state. Only menu items
-    to which the currently logged in user has access will appear
+-   Drop-down elements on the left side provide links to product lines
+    and never include a secondary CTA.
+
+-   Drop-down elements on the left side do not have a defined disabled
+    state. The user only sees menu items to which they have access.
 
 {% endcolumn %}
 {% column right:"col-md-offset-1 col-md-7" %}
-{% figure [caption:"Eyebrow (leftside) dropdown specs"] [class:"image bg-light border"] %}
+{% figure [caption:"Eyebrow (left side) drop-down element specs"]
+[class:"image bg-light border"] %}
 ![]({{site.cdn_url}}/img/layout/eye-left-drop.svg)
 {% endfigure %}
 {% endcolumn %}
 </div>
 
 ## Eyebrow (right side)
 
-The right side of the eyebrow menu is comprised of the user name section and
+The right side of the eyebrow menu shows the user name section and
 platform level services button groups.
+
 <div class="row">
 {% column left:"col-md-4" %}
 
--   Platform level services are any services that exist across multiple product
-    lines. Ex: Account Settings, Notifications, Ticketing, etc.
+-   Platform level services are any services that exist across
+    multiple product lines, such as account settings, notifications,
+    and ticketing.
 
--   The user name section and each service button group is separated by a
-    section divider
+-   A section divider separates the user name section and each service
+    button group.
 
 -   Service groups
 
-    -   User Settings
+    -   User settings
 
-    -   Account Management
+    -   Account management
 
         - Account
         - Billing
 
-    -   Support Management
+    -   Support management
 
-        - Messages / Notification
+        - Messages / notification
         - Tickets
         - Support
 
 {% endcolumn %}
 {% column right:"col-md-offset-1 col-md-7" %}
-{% figure [caption:"Eyebrow (rightside) construction specs"] [class:"image bg-light border"] %}
+{% figure [caption:"Eyebrow (right side) construction specs"] [class:"image bg-light border"] %}
 ![]({{site.cdn_url}}/img/layout/global-layer/eyebrow-right.svg)
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-### Eyebrow (right side) Dropdowns
+### Eyebrow (right side) drop-down elements
 
 <div class="row">
 {% column left:"col-md-4" %}
 
--   Items within the right-side of the eyebrow produce either a dropdown or
-    display an off-canvas panel onClick
+-   Items within the right side of the eyebrow show either a drop-down
+    menu or an off-canvas panel onClick.
```

Comment:
no. I believe onClick is how one defines the trigger type that initiates the defined behavior. we can separate that into regular language to on mouse click if @technabors doesn't object.